src/client/pps/pps.proto

```diff
@@ -34,6 +34,13 @@ message Transform {
   string user = 10;
   string working_dir = 11;
   string dockerfile = 12;
+  BuildSpec build = 15;
+}
+
+message BuildSpec {
+  string path = 1;
+  string language = 2;
```

Comment:
I'm guessing that language is meant to add some extra ergonomics to this feature by giving commonly used languaes prefab build containers. What's actually in those images, do they include Pachyderm clients in addition to the standard language containers?

Reply:
The builders section of the PR description gives some outline, and you can see the official builders in `etc/pipeline-build`. In short, the Pachyderm client isn't included at the moment (though maybe it should be?) Rather, it's just the language's official docker images, with some glue shell scripts that are expected to execute in the build and run phases.